index.html

```diff
@@ -522,8 +522,18 @@ <h5 class="card-title">Word Count Calculator</h5>
       <a href="Public/word-count-calculator.html" class="btn btn-outline-primary">Try Now</a>
     </div>
   </div>
-  <!--This div is for Library website-->
+  <!--This div is for crypto-app website-->
   <div class="card card1">
+    <img class="card-img-top" src="assets/Images/crypto.jpg" alt="Library">
+   <div class="card-body">
+      <h5 class="card-title">Crypto-App</h5>
+      <p>Know the prices and many more about crypto!</p>
+      <a href="Public/crypto_app.html" target="_blank" class="btn btn-outline-primary">Try Now</a>
+    </div>
+  </div>
+  <!--End of crypto-app website-->
+   <!--This div is for Library website-->

```

Comment:
**Make this correct.**

![Screenshot 2022-03-06 162642](https://user-images.githubusercontent.com/73425223/156920121-5785846b-d284-47af-9d9e-e0e565ef2b7c.png)